.gitignore

```diff
@@ -11,3 +11,4 @@
 /.idea/
 /OnLoop/OnLoop.iml
```

Comment:
@maria-salmasian jan you can write *.iml which will mean all iml files , also you can write the same you can do for the class files

Reply:
@maria-salmasian I dont see the fix

Reply:
done